zanata-war/src/main/resources/messages.properties

```diff
@@ -635,9 +635,20 @@ jsf.iteration.NoDocumentInVersion=No documents in this version
 jsf.iteration.NoLanguagesInVersion=No language in this version
 jsf.iteration.TranslateOnline=Translate Online
 jsf.iteration.ViewOnline=View Online
-
-
-
+jsf.iteration.mergeTrans.header=Merge translations from an existing version
+jsf.iteration.mergeTrans=Merge Translations
+jsf.iteration.mergeTrans.description1=All <dfn title='Translation which has identical language, source string, document name.'>matching</dfn> <strong><span class='txt--success'>translated</span>/<span class='txt--highlight'>approved</span></strong> translations from the source version will be copied to the target version.
+jsf.iteration.mergeTrans.description2=If there is an <strong>existing <span class='txt--success'>translated</span>/<span class='txt--highlight'>approved</span></strong> translation, the <strong>newer</strong> translation will be used.
+jsf.iteration.mergeTrans.toVersion.label=To <span class="heading--secondary">(Target)</span>
+jsf.iteration.mergeTrans.fromVersion.label=From <span class="heading--secondary">(Source)</span>
+jsf.iteration.mergeTrans.keepExistingTranslated=Keep <strong>existing <span class='txt--success'>translated</span>/<span class='txt--highlight'>approved</span></strong> strings
+jsf.iteration.mergeTrans.keepExistingTranslated.Description=Don't replace existing translated/approved translations with newer translations from this source.
```

Comment:
I have heard from docs people that it is better not to use contractions - i.e. "Do not replace..."